### PR TITLE
Update nanoflann to v1.5.5

### PR DESCRIPTION
**Pull request by rjanvier**
fix an annoying cmake warning.

### CODE REVIEW

File: src/Kriging.cpp [hunk 1]
```diff
@@ -3,876 +3,790 @@
...
 
-//! Cholesky matrix decomposition to lower triangular matrix and its conjugate transpose
+//! Cholesky matrix decomposition to lower triangular matrix and its conjugate
+//! transpose
 /** Restricted to positive-definite matrices
-**/
-class CholeskyDecomposition
-{
+ **/
+class CholeskyDecomposition {
```
Comment (dgirardeau) [inline src/Kriging.cpp:21]: The convention in CC's code is still to keep '{' and '}' on separate lines 😉

Comment (dgirardeau) [reply src/Kriging.cpp:21]: Here and below!

File: src/Kriging.cpp [hunk 2]
```diff
@@ -3,876 +3,790 @@
...
+    }
+
+    return y;
+  }
 
 protected:
-
-	//! Decomposed matrix
-	Kriging::Matrix m_matrix;
+  //! Decomposed matrix
```
Comment (dgirardeau) [inline src/Kriging.cpp:97]: And has the spacing changed as well? (should be tabs)

**Review by dgirardeau — CHANGES_REQUESTED** [review]
It's hard to see what the real changes are (with all these syntax changes). Can you revert them?

File: src/Kriging.cpp (hunk 2 above)
Comment (rjanvier) [reply src/Kriging.cpp]: it should be a bit more readable now!

**Review by dgirardeau — APPROVED** [review]
Ah ah, much more readable indeed

**Pull request merged**